client/src/app/core/repositories/agenda/agenda-item-repository.service.ts

```diff
@@ -128,13 +128,17 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
     }
 
     public async addItemToAgenda(contentObject: BaseViewModel & HasAgendaItem): Promise<Identifiable> {
-        return await this.create({ id: contentObject.id } as AgendaItem);
+        return await this.actions.sendRequest(ActionType.AGENDA_ITEM_CREATE, {
+            content_object_id: fqidFromCollectionAndId(contentObject.collection, contentObject.id),
```

Comment:
This should work: `contentObject.fqid`. Plase have a look in `base-model.ts` and use your new function there (In the file it is still hard-coded).